web/src/app/shared/services/classifications/classification-categories.service.ts

```diff
@@ -2,87 +2,36 @@ import { HttpClient } from '@angular/common/http';
 
 import { Injectable } from '@angular/core';
 import { environment } from 'environments/environment';
-import { Observable, ReplaySubject, BehaviorSubject } from 'rxjs';
-import { CustomFieldsService } from 'app/services/custom-fields/custom-fields.service';
-import { Pair } from 'app/models/pair';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Customisation } from '../../../models/customisation';
+
+const customisationUrl = 'environments/data-sources/taskana-customization.json';
+
+export interface CategoriesResponse { [key: string]: string[] }
 
 @Injectable()
 export class ClassificationCategoriesService {
```

Comment:
This name is not correct anymore. We should rename it and think about a way to restructure our services

Reply:
=> ticket

Reply:
ich machs.